src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1318,25 +1340,45 @@ def download_from_original_stable_diffusion_ckpt(
     # Convert the UNet2DConditionModel model.
     unet_config = create_unet_diffusers_config(original_config, image_size=image_size)
     unet_config["upcast_attention"] = upcast_attention
-    unet = UNet2DConditionModel(**unet_config)
+    with init_empty_weights():
```

Comment:
we probably need to qualify this with `if is_accelerate_available():` check if we don't have accelerate